tests/acceptance/expected-failures-on-S3NG-storage.md

```diff
@@ -1368,6 +1368,7 @@ _ocs: api compatibility, return correct status code_
 
 ### [send PUT requests to another user's webDav endpoints as normal user](https://github.com/owncloud/ocis/issues/2893)
 - [apiAuthWebDav/webDavPUTAuth.feature:58](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiAuthWebDav/webDavPUTAuth.feature#L58)
+- [apiAuthWebDav/webDavPUTAuth.feature:70](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiAuthWebDav/webDavPUTAuth.feature#L70)
```

Comment:
the tests expect  403, but ocis/reva return 404 by design, to not leak the existence of files/spaces.